pyomo/contrib/pyros/tests/test_uncertainty_sets.py

```diff
@@ -1886,6 +1894,42 @@ def test_normal_construction_and_update(self):
         np.testing.assert_allclose(
             new_scale, eset.scale, err_msg="EllipsoidalSet scale update not as expected"
         )
+        np.testing.assert_allclose(
+            # evaluate chisquare CDF for 2 degrees of freedom
+            # using simplified formula
+            1 - np.exp(-new_scale / 2),
+            eset.gaussian_conf_lvl,
+            err_msg="EllipsoidalSet Gaussian confidence level update not as expected",
+        )
+
+    def test_normal_construction_and_update_gaussian_conf_lvl(self):
+        """
+        Test EllipsoidalSet constructor and setter
+        work normally when arguments are appropriate.
+        """
+        init_conf_lvl = 0.95
+        eset = EllipsoidalSet(
+            center=[0, 0, 0],
+            shape_matrix=np.eye(3),
+            scale=None,
+            gaussian_conf_lvl=init_conf_lvl,
+        )
+
+        self.assertEqual(eset.gaussian_conf_lvl, init_conf_lvl)
+        np.testing.assert_allclose(
+            sp.stats.chi2.isf(q=1 - init_conf_lvl, df=eset.dim),
+            eset.scale,
+            err_msg="EllipsoidalSet scale not as expected",
```

Comment:
Suggestion: I won't have this hold up the PR, but it might be worth making this message into its own class-level var so you don't have to potentially change the error message in a bunch of places if you decide to modify the language or the error message someday.